snorkel/classification/models/advanced/advanced_classifier.py

```diff
@@ -357,3 +360,77 @@ def load(self, model_path: str):
 
         logging.info(f"[{self.name}] Model loaded from {model_path}")
         self._move_to_device()
+
+    @classmethod
+    def from_modules(
+        self,
+        modules: List[nn.Module],
+        dropout: float = 0.0,
+        loss_func: Callable[..., torch.FloatTensor] = ce_loss,
+        output_func: Callable[..., np.ndarray] = softmax,
+        metrics: List[str] = ["accuracy"],
+        task_name: str = "task",
+        data_name: str = "data",
+        **kwargs,
+    ):
+        """Instantiates a simple AdvancedClassifier from `Module`s instead of `Task`s
+        This is intended as a convenience method for quickly creating an
+        AdvancedClassifier in certain situations where many defaults can be inferred.
+        Only use this method of instantiation if all of the following properties apply:
+        - Your problem has only one task
+            - That task name is the only key in dataloader.task_to_label_dict
+        - Your data has only one field
+            - The `data_name` you pass here and that field name in the X_dict match
+        - Your architecture is a simple sequential path using each Module exactly once
+        - You don't require passing any custom metric functions to the Scorer
```

Comment:
Even after reading this doc string I have no idea what's going on here, when this method is applicable, how to use it... way to complex for a simple single task classifier.

Of course, I haven't reviewed this new MTL architecture- but that's *exactly* why I think my take is valuable here. Because if it takes intense insider knowledge of some MTL model "task flow" thing to understand how to use a basic single task classifier class in our framework, then we're done- no one's going to use it.